README.md

```diff
@@ -22,31 +22,36 @@ Prerequisites
  * GovUK and MOJ design systems
  * Nunjucks (UI templates)
 
+### Getting started
+
+* Clone this repo
+* Run `nvm install` to ensure you are using the correct version of node
+* Create `.env` files for each service. You can use `.env.sample` as a base. Some sensitive variables need to be shared from the team
+* Generate JWT keypairs with `secrets/set_jwt_keypair.sh`
```

Comment:
on Windows you may need to run it as `bash secrets/set_jwt_keypair.sh`

Reply:
updated